SpatialGDK/Documentation/content/troubleshooting.md

```diff
@@ -37,7 +37,7 @@ You're building against an unsupported version of Unreal Engine. Make sure you'r
 
 ### Q: My game uses reliable multicast RPCs - why does the SpatialOS GDK for Unreal not support these?
 
-The underlying implementation of multicast RPCs uses SpatialOS [events](https://docs.improbable.io/reference/latest/shared/glossary#event). SpatialOS events can only be sent unreliably. Additionally, the cost of a multicast RPC scales with the number of client-workers present in a deployment, which means they can get very expensive. A better approach would be to send RPCs to only the workers that are close to the broadcasting worker.
+The underlying implementation of multicast RPCs uses SpatialOS [events](https://docs.improbable.io/reference/latest/shared/glossary#event). SpatialOS events can only be sent unreliably. Additionally, the cost of a multicast RPC scales with the number of client-worker instances present in a deployment, which means they can get very expensive. A better approach would be to send RPCs to only the worker instances that are close to the broadcasting worker instance.
```

Comment:
What is the "broadcasting worker instance" and what does "close" mean here?
Shall we make a ticket to go through this 1.1+?

Reply:
Hmm, yeah maybe. I thought the troubleshooting guide was on our 1.0 radar at one point, but could be wrong. Not sure how important it is for 1.1.

For this PR, I only reviewed the new stuff, as the existing stuff looked a bit can-of-worms-y...